### DISCUSSION TITLE
How do you configure mathjs in the CLI?

### DISCUSSION THREAD

**Discussion by radianrad** [josdejong/mathjs#3026] [Q&A] [answered]
Hello! I have several custom functions and datatypes that I import typically using math.config. I'd like to also use these in the repl/CLI. Is that possible with flags or something or do I need to edit those files to include that?

Thank you

**Comment by josdejong** [comment, marked as answer]
That is a good question.

In the repl `./bin/repl.js` you are just running javascript, so you can use `math.import(...)` to import your own functions and data types
For the cli `./bin/cli.js` there is no such option, and you will have to edit the source code of it.

It would be interesting to think through command line options for the CLI to configure mathjs and import functions. Anyone interested picking that up?

**Comment by radianrad** [comment]
Thank you! I can find a workaround there.
